Daytime-Operations/Auxiliary-Telescope/AT-Calibrations/LATISS-Combined-Calibrations-Procedure/latiss-combined-calibrations-procedure.rst

```diff
@@ -173,7 +178,61 @@ After loading the script, a window that contains two sections, ``SCHEMA`` (top)
 - ``oods_timeout``: Timeout value, in seconds, for the Observatory Operations Data Service (``OODS``).
   Default: ``120``.
 
-An example set of configuration parameters is as follows:
+
+Configuration examples
+----------------------
+
+**Preferred daily script mode to be run**: if no configuration parameters are passed to LOVE and the default parameters are used, the script will take 21 biases, 21 darks of 5 seconds each one, and 21 flats of 5 seconds each one. In each case, the first image will be discarded. New combined calibrations will not be generated, and verification of the images taken will be performed using the existing combined calibrations in the ``LATISS/calib`` collection (i.e., the script will do ``external verification``). In this case, no defects will be made. Following DMTN-222, a gain estimate will be produced from each of the 10 flat pairs taken. **Users should adjust parameters when needed, for example, the exposure times or the number of exposures taken**.
+
+
+
+
+If the exposure times need to change, it can be done as follows:
+
+.. code-block:: text
+    
+    exp_times_dark: 20
+    exp_times_flats: 30
+
+If both the number of exposures and exposure times need to change, it can be done like this:
+
+.. code-block:: text
+
+    n_bias: 30
+    n_dark: 5
+    exp_times_dark: [5, 10, 15, 20, 25]
+    n_flat: 10
+    exp_times_flat: [5, 10, 15, 20, 25, 30, 35, 40, 45, 50]
```

Comment:
n_flat is 10 flats for each exposure time, yes?

It would be useful to have `exp_times_flat` have a length that is not equal to 10 to show that. Same with the darks

Reply:
> n_flat is 10 flats for each exposure time, yes?

No. It is the total number of flats to take. If then ``exp_times_flat`` is just a single number, say 35, then it will take 10 flats of 35 seconds for each one. But if you want different exposure times for the flats, then an array with the same length as the number of flats must be specified (as in the example).

Reply:
hmm, ok, so I think the right way to implement this is to add more logic into the configuration.
Basically if `exp_times_dark` is a single value, then n_dark=5 takes 5 exposures, if a list is specified, then it takes 5 exposures for each element in the list.
Maybe best to leave this for a future ticket.